### PR TITLE
Make data sharing disclosure a little clearer

### PR DESCRIPTION
**Pull request by yjernite**
Quick proposed update so the disclosure draws the eye a little more - currently it's easy to miss

image

### CODE REVIEW

**Comment by yjernite** [comment]
Simple rephrasing to start the conversation, but I'd love to hear from @ImaBello how the text could be improved!

File: src/lib/components/LoginModal.svelte [hunk 1]
```diff
@@ -33,7 +33,8 @@
 			misinformation. Do not use this application for high-stakes decisions or advice.
 		</p>
 		<p class="px-2 text-sm text-gray-500">
-			Your conversations will be shared with model authors unless you disable it from your settings.
+			⚠️ <b>Data sharing:</b> Your conversations will be shared with the authors of the model
+			you use unless you disable data sharing in the HuggingChat settings.
```
Comment (julien-c) [inline src/lib/components/LoginModal.svelte:37]:
```suggestion
   Your conversations will be shared with model authors unless you disable data sharing in the HuggingChat settings.
```

**Comment by ImaBello** [comment]
I suggest:

"By default, your conversations are shared with the authors of the model to improve their training data and model. You can disable data sharing in the HuggingChat settings. See this [privacy note ](https://huggingface.co/chat/privacy)for further details."

cc @annatrdj for visibility

**Comment by annatrdj** [comment]
Thanks for the tag Ima! I actually prefer the initial language :) 
1/ "unless you disable" makes it clear that they *have* to do something to stop data sharing
2/ no mention of the purpose to keep it vague (and in case data sharing is for another purpose)

**Pull request closed without merging**